### DISCUSSION TITLE
Robust Visual SLAM Using MATLAB Mobile Sensor Streaming (Project 213)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#44] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 213](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Robust%20Visual%20SLAM%20Using%20MATLAB%20Mobile%20Sensor%20Streaming)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by zorjo** [comment]
Hi I am a student of NIT Durgapur and interested in working and collaborating on this project

**Comment by adityav2000** [comment]
Hi I am a student of MNIT Bangalore and interested in working and collaborating on this project

**Reply by zorjo** [reply]
Would  you be interested in working collaboratively together?

**Comment by adityav2000** [comment]
Yeah sure

**Reply by zorjo** [reply]
well, you can email me at sourjyo@protonmail.com

**Comment by zorjo** [comment]
can you use videos instead of imagedatastore in order to make vSLAM?
the IndexImages requires an imagedatastore object

**Comment by adityav2000** [comment]
Yeah, I can do that. But I don't have much idea about this project, so i
need to know everything from the basics.

**Reply by zorjo** [reply]
how about the Indeximages function requiring a imds instead of video? what to do then?

**Comment by adityav2000** [comment]
I didn't get that...can you please elaborate

**Reply by zorjo** [reply]
I'm using this code as a starting point
https://in.mathworks.com/help/vision/ug/monocular-visual-simultaneous-localization-and-mapping.html
this code is in the loop closure part of the code
`loopDatabase    = indexImages(subset(imds, loopCandidates), bofData.bof);`
here imds is an imagedatastore object which is a folder of images 
however i have a video to use this on
hence the problem

**Comment by adityav2000** [comment]
Awesome, thanks! Could you please send the video too....it would be really
helpful.

**Reply by zorjo** [reply]
by video i mean i have  recorded a video to perform vSLAM on instead of downloading a dataset of images.
Any video will do, so I doubt it will be any useful

**Comment by Jaskine16** [comment]
Hi I am a student at University of the Philippines-Baguio and interested to work and collaborate on this project.

**Reply by nidhi906** [reply]
hi, i am doctoral student. anybody want to collaborate to this project?

**Comment by Jaskine16** [comment]
Sure.
